frontend/src/static/css/style.scss

```diff
@@ -115,6 +115,50 @@ header {
       margin-bottom: 1em;
       margin-left: 1em;
     }
+
+    .error-message-box {
+      background-color: #F8D7DA;
```

Comment:
Please use custom class `mui-color` instead of specifying the color on your own.